data/json/recipes/recipe_others.json

```diff
@@ -7904,5 +7904,36 @@
     [ [ "wheel", 1 ] ],
     [ [ "motor", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "neoprene",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_OTHER",
+  "skill_used": "cooking",
+  "skills_required": [ "cooking", 4 ],
+  "difficulty": 4,
+  "time": 25000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [["adv_chemistry", 3] , ["textbook_chemistry", 3]],
+  "qualities":[
+    {"id":"CHEM","level":2,"amount":1}
+  ], "tools": [
+    [
+      [ "hotplate", 100 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 100 ],
+      [ "fire", -1 ],
+      [ "chemistry_set", 100]
```

Comment:
You'll need a chemistry set anyway, because recipe needs an item with CHEM quality of level 2. So, having chemistry set as a tool there just clutters the tool list.